File: RepoTransBench/target_projects/Java/C#/spengilley_ActivityFragmentMVP/public_tests/ModulesPublicTest.cs
using Xunit;
using ActivityFragmentMVP;

namespace ActivityFragmentMVP.Tests.Public
{
    public class ModulesPublicTest
    {
        [Fact]
        public void list_nonNullDifferentAppInstance()
        {
            App testApp = new App();
            object[] modules = Modules.list(testApp);
            Assert.NotNull(modules);
            Assert.True(modules.Length > 0);
            Assert.IsType<AppModule>(modules[0]);
        }
    }
}

File: RepoTransBench/target_projects/Java/Go/spengilley_ActivityFragmentMVP/public_tests/modules_public_test.go
package public_tests

import (
	"testing"
	"github.com/spengilley/activityfragmentmvp/tests/original"
)

func TestModules_ListNonNullDifferentAppInstance(t *testing.T) {
	testApp := &original.App{}
	modules := original.Modules{}.list(testApp)
	if modules == nil {
		t.Fatal("Expected Modules.list() to return a non-nil slice")
	}
	if len(modules) == 0 {
		t.Error("Expected Modules.list() to return a non-empty slice")
	}
	if _, ok := modules[0].(*original.AppModule); !ok {
		t.Errorf("Expected modules[0] to be of type *AppModule")
	}
}